autoscale-email-alert-dispatcher/pom.xml

```diff
@@ -33,17 +33,24 @@
         <dependency>
             <groupId>com.github.autoscaler</groupId>
             <artifactId>autoscale-api</artifactId>
-            <version>1.5.0-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
The version number should be in the dependency management section in the aggregator pom, and in that location it is preferable to hard-code the version number rather than using `${project.version}`.